src/docs/labels/label-fqn/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Label-fqn
```

Comment:
The title should be human readable, i.e. `Label FQN`.
Also, let's also put in brackets what `fqn` means.